open-sphere-plugins/feature-actions/src/main/java/io/opensphere/featureactions/controller/StyleFactory.java

```diff
@@ -355,37 +346,38 @@ private DynamicEllipseFeatureVisualization newEllipseStyle(DataTypeInfo dataType
         return style;
     }
 
-    /**
-     * Adds label style info to a copy of the given style.
+    /* Adds label style info to a copy of the given style.
      *
      * @param labelAction the LabelAction
+     * 
      * @param dataType the data type
+     * 
      * @param style the style to add to
-     * @return the style
-     */
-    private FeatureVisualizationStyle addLabelStyle(LabelAction labelAction, DataTypeInfo dataType,
-            FeatureVisualizationStyle style)
-    {
-        LabelOptions labelOptions = labelAction.getLabelOptions();
-
-        List<String> labels = New.list();
-        ColumnLabels columnLabels = labelOptions.getColumnLabels();
-        if (columnLabels.isAlwaysShowLabels())
-        {
-            for (ColumnLabel columnLabel : columnLabels.getColumnsInLabel())
-            {
-                List<String> args = Arrays.asList(Boolean.toString(columnLabel.isShowColumnName()), columnLabel.getColumn());
-                labels.add(StyleUtils.listSupp.writeList(args));
-            }
-        }
-        style.setParameter(AbstractFeatureVisualizationStyle.LABEL_COLUMN_KEY_PROPERTY_KEY, labels, this);
-        style.setParameter(AbstractFeatureVisualizationStyle.LABEL_ENABLED_PROPERTY_KEY, Boolean.TRUE, this);
-        style.setParameter(AbstractFeatureVisualizationStyle.LABEL_COLOR_PROPERTY_KEY, labelOptions.getColor(), this);
-        style.setParameter(AbstractFeatureVisualizationStyle.LABEL_SIZE_PROPERTY_KEY, Integer.valueOf(labelOptions.getSize()),
-                this);
-
-        return style;
-    }
+     * 
+     * @return the style */
+//    private FeatureVisualizationStyle addLabelStyle(LabelAction labelAction, DataTypeInfo dataType,
+//            FeatureVisualizationStyle style)
+//    {
+//        LabelOptions labelOptions = labelAction.getLabelOptions();
+//
+//        List<String> labels = New.list();
+//        ColumnLabels columnLabels = labelOptions.getColumnLabels();
+//        if (columnLabels.isAlwaysShowLabels())
+//        {
+//            for (ColumnLabel columnLabel : columnLabels.getColumnsInLabel())
+//            {
+//                List<String> args = Arrays.asList(Boolean.toString(columnLabel.isShowColumnName()), columnLabel.getColumn());
+//                labels.add(StyleUtils.listSupp.writeList(args));
+//            }
+//        }
+//        style.setParameter(AbstractFeatureVisualizationStyle.LABEL_COLUMN_KEY_PROPERTY_KEY, labels, this);
+//        style.setParameter(AbstractFeatureVisualizationStyle.LABEL_ENABLED_PROPERTY_KEY, Boolean.TRUE, this);
+//        style.setParameter(AbstractFeatureVisualizationStyle.LABEL_COLOR_PROPERTY_KEY, labelOptions.getColor(), this);
+//        style.setParameter(AbstractFeatureVisualizationStyle.LABEL_SIZE_PROPERTY_KEY, Integer.valueOf(labelOptions.getSize()),
+//                this);
+//
+//        return style;
+//    }

```

Comment:
I believe we need to keep this and I don't think this is the cause of the ticket.